api/src/test/java/org/pmiops/workbench/api/WorkspacesControllerTest.java

```diff
@@ -260,6 +260,8 @@
   @Autowired private CohortAnnotationDefinitionController cohortAnnotationDefinitionController;
   @Autowired private WorkspacesController workspacesController;
 
+  @MockBean FreeTierBillingService mockFreeTierBillingService;
```

Comment:
Because FreeTierBillingService now has a FreeTierAuditor dependency, this class needs to `@Import` it to run.  Instead, I took the opportunity to change this from a sorta-mock to a proper mock.